src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/KeywordCompletionProviderTests.vb

```diff
@@ -173,5 +173,23 @@ End Class
 
             VerifyNoItemsExist(code)
         End Sub
+
+        <WorkItem(4167, "https://github.com/dotnet/roslyn/issues/4167")>
+        <Fact(), Trait(Traits.Feature, Traits.Features.KeywordRecommending)>
+        Public Sub ImplementsAfterSub()
```

Comment:
There don't appear to be any obviously named tests for any ISyntaxFactsService methods. I assume testing just goes through regular features.